guide/13-managing-arcgis-applications/hub-submodule-intro.ipynb

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
<ul><li>I think the first sentence should explicitly mention ArcGIS Entperise Sites:</li><li class="ql-indent-1"><em>The&nbsp;</em><code><em>hub</em></code><em>&nbsp;submodule within&nbsp;</em><code><em>arcgis.apps</em></code><em>&nbsp;acts as the pythonic interface to </em><a href="https://www.esri.com/en-us/arcgis/products/arcgis-hub/overview" target="_blank">ArcGIS Hub</a> or [ArcGIS Enterprise Sites](https://enterprise.arcgis.com/en/sites/latest/get-started/about-this-application.htm).</li><li>The appropriate relative links should be added to this page:</li><li class="ql-indent-1">Hub Basic - [Hub Basic](../guide/hub-for-guide-basic)</li><li class="ql-indent-1">Hub Premium - <a href="../guide/hub-for-guide-premium" target="_blank">Hub Premium</a></li><li class="ql-indent-1">Enterprise Sites - [Enterprise Sites[(../guide/enterprise-sites)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>